lightningnetwork/lnd
# docs/comprehensive lncli

## Change Description
Adds necessary tags to make all currently available lncli commands known to the [API documentation tool](https://github.com/lightninglabs/lightning-api-ng). Fixes #8102 

NOTE: this does not add any sort of checker to guarantee that these tags are always added. Such a facility would likely require a golang AST library and I'm not sure it's worth the work.

## Steps to Test
Build the API documentation against this branch/commit and see if all available commands appear in the docs. I did this by hand. Doing this automatically would also require an AST traversing library.

## Pull Request Checklist
### Testing

### Code Style and Documentation

📝 Please see our  [Contribution Guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md) for further guidance.
